index.html

```diff
@@ -1,271 +1,325 @@
-          <button class="card-button btn-cards">See Project</button>
+  
+        <div class="form-control">
+          <input type="text" placeholder="Email address" id="email" />
+          <div class="input-line"></div>
         </div>
-      </div>
-    </article>
-  </section>
-  <!-- About-us section -->
-  <section id="about-us" class="about-us">
-    <div class="about-me">
-      <div class="about-us-header">
-        <h1>About me</h1>
-      </div>
-      <div class="about-us-text">
-        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my
-          work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-        </p>
-      </div>
-    </div>
-    <button class="btn-about">Get My Resume</button>
-    <div class="group3"></div>
-    <div class="about-us-border"></div>
-    <div class="ellipse1"></div>
-    <div class="about-skill">
-      <ul class="about-skill-item">
-        <li class="about-skill-items">
-          <p class="skill-language">
-            <i><img src="assets/images/Rectangle2.svg" alt="" width="17"></i>
-            <strong class="language">Language</strong>
-          </p>
-          <ul class="margin-top-10">
-            <li class="language-items">Javascript</li>
-            <li class="language-items">Ruby</li>
-            <li class="language-items">Html</li>
-            <li class="language-items">Css</li>
-          </ul>
-        </li>
-        <li class="about-skill-items">
-          <p class="skill-language">
-            <i><img src="assets/images/Rectangle3.svg" alt="" width="17"></i>
-            <strong class="language  margin-05">Frameworks</strong>
-          </p>
-          <ul class="margin-top-10">
-            <li class="language-items">Bootstrap</li>
-            <li class="language-items">Ruby on Rails</li>
-            <li class="language-items">RSPec</li>
-            <li class="language-items">CapyBara</li>
-            <li class="language-items">Selenium</li>
-          </ul>
-        </li>
-        <li class="about-skill-items">
-          <p class="skill-language">
-            <i><img src="assets/images/Rectangle1.svg" alt="" width="17"></i>
-            <strong class="language">Skills</strong>
-          </p>
-          <ul class="margin-top-10">
-            <li class="language-items">Codekit</li>
-            <li class="language-items">Github</li>
-            <li class="language-items">Codepen</li>
-            <li class="language-items">Gitlab</li>
-            <li class="language-items">Terminal</li>
-          </ul>
-        </li>
-      </ul>
-    </div>
-  </section>
-</body>
-
-</html>
\ No newline at end of file
+  
+        <div class="form-control">
+          <textarea placeholder="Enter text here" id="message" cols="30" rows="7"></textarea>
+        </div>
+        <button type="submit" class="btn-get">Get It Touch</button>
+      </form>
+     </div> 
```

Comment:
- [ ] It would be nice to include a `maxlength="500"` attribute for the **_textarea_** form control as specified in the project requirements. Implementing this would ensure that the number of characters inputted in the message field would not exceed 500. 😄 
- [ ] Another major requirement for this milestone is that all form inputs should have the attribute `required` on them. It would be nice to implement this requirement because it would ensure that users can not submit the form without filling in all necessary fields.